.venv/bin/mqtt_check.py

```
@@ -0,0 +1,89 @@
+    parser = argparse.ArgumentParser(add_help=False,
+                                     description="MQTT login check")
+    parser.add_argument("--help", action="help", help='show this help message and exit')
+    parser.add_argument('target', action='store', help='[[domain/]username[:password]@]<targetName>')
+    parser.add_argument('-client-id', action='store', help='Client ID used when authenticating (default random)')
+    parser.add_argument('-ssl', action='store_true', help='turn SSL on')
+    parser.add_argument('-port', action='store', default='1883', help='port to connect to (default 1883)')
+    parser.add_argument('-debug', action='store_true', help='Turn DEBUG output ON')
+
+    try:
+        options = parser.parse_args()
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)